reactioncommerce/reaction
# Fixes #2810: /shop added to URL

Resolves #2810  
Impact: minor  
Type: bugfix

## Issue
The route returned was not prefixed with `/shop` hence was resulting in a 404 page

## Solution
Added `/shop` to the route being returned. This is done by added a param `/shopPrefix` to the marketplace settings.


## Testing
1. Configure shop to take order (configure shipping/payment)
1. Place an order
1. Return to home page
1. Click on the "bell" icon in the top nav
1. Click on the "Your Order was accepted" link
1. Observe that you get to the notifications page.

I am not sure the notifications page is where you are supposed to land but from the code it looks like that was the intended behaviour.
